src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -8632,7 +8632,7 @@ static void Main()
 
             Assert.IsType<TupleElementFieldSymbol>(m1Item1);
             Assert.IsType<TupleFieldSymbol>(m2Item1);
-            Assert.IsType<TupleRenamedElementFieldSymbol>(m2a2);
+            Assert.IsType<TupleVirtualElementFieldSymbol>(m2a2);
```

Comment:
The tests seems to have a lot of IsType checks. It seems to be designed to validate that fields are represented by particular types - TupleElementFieldSymbol, TupleFieldSymbol, TupleVirtualElementFieldSymbol.

Not sure if all these chacks can be expressed in terms of IsXXXX methods. Let's see...